_opensearch/query-dsl/index.md

```diff
@@ -12,16 +12,49 @@ redirect_from:
 
 # Query DSL
 
-While you can use HTTP request parameters to perform simple searches, you can also use the OpenSearch query domain-specific language (DSL), which provides a wider range of search options. The query DSL uses the HTTP request body, so you can more easily customize your queries to get the exact results that you want.
+OpenSearch provides a query domain-specific language (DSL) that you can use to perform customized searches on your data.
 
-For example, the following request performs a simple search to search for a `speaker` field that has a value of `queen`.
+Query DSL provides several query categories. You can even perform more complicated searches by using different elements from each variety to find more granular data that you need.
+
+The following query DSL categories include:
+
+- **Compound boolean queries** – To search with clause rules that act as logical operators to combine several simple query terms.
```

Comment:
Even though we don't have all of the documentation yet, should we list all query types here and not only the ones for which we do have documentation?

Reply:
I agree. Let's provide a list of the ones that we support. We can always link to the documentation for each one later.

Reply:
We would need those sections to be available on the site. This is a tech doc, not marketing "what's coming" announcement.

Reply:
ok will do. 
The reason I didn't list them here is because it would be confusing to see it called out in the overview, and then customers can't go to the missing section.